astropy/modeling/functional_models.py

```diff
@@ -33,6 +35,17 @@ class BaseGaussian1D(Fittable1DModel):
     mean = Parameter(default=0)
     stddev = Parameter(default=1)
```

Comment:
Just out of curiosity, why not simply set ``Parameter(default=1, bounds=(FLOAT_EPSILON, None))`` here - is it because the user then can't override the bounds?